beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -99,15 +99,18 @@ func (vs *Server) GetBeaconBlock(ctx context.Context, req *ethpb.BlockRequest) (
 	}
 
 	resp, err := vs.BuildBlockParallel(ctx, sBlk, head, req.SkipMevBoost, builderBoostFactor)
-	log.WithFields(logrus.Fields{
+	log := log.WithFields(logrus.Fields{
 		"slot":               req.Slot,
 		"sinceSlotStartTime": time.Since(t),
 		"validator":          sBlk.Block().ProposerIndex(),
-		"err":                err,
-	}).Info("Finished building block")
+	})
+
 	if err != nil {
+		log.WithError(err).Error("Finished building block")
```

Comment:
Should the error path still say `Finished building block`? I think something like `Could not build block`/`Failed to build block` would be more appropriate

Reply:
I take it back. The log level makes it clear there was an error